### DISCUSSION TITLE
Maximum value of ActiveDeadlineSeconds

### DISCUSSION THREAD

**Discussion by SasaluckTWxMotional** [argoproj/argo-workflows#8299] [Q&A] [answered]
What is the maximum value of ActiveDeadlineSeconds ?

**Comment by terrytangyuan** [comment, marked as answer]
I don't think Argo Workflows has any limit on this so it should behave the same way as `int64`.

**Reply by SasaluckTWxMotional** [reply]
@terrytangyuan I see, thank you very much!
